app/presenters/admin/user_presenter.rb

```diff
@@ -0,0 +1,48 @@
+class Admin::UserPresenter < SimpleDelegator
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers
+  include ActionView::Helpers::TranslationHelper
+  include ActionView::Context
+
+  def initialize(user, view)
+    @user = user
+    @view = view
+    super(@user)
+  end
+
+  def admin_actions(user)
+    content_tag :div, class: 'user_options btn_group pull-right' do
```

Comment:
What do you think about creating partials for that instead of using `content_tag` and stuff like that and the partial would be coupled with the presenter? It would gives us HTML syntax highlight and be easier to maintain 😄